### PR TITLE
New design for License Page

### PR DESCRIPTION
**Pull request by Alevsk**
## How it looks

Screen Shot 2021-02-18 at 19 16 36

## Mockup

https://xd.adobe.com/view/b3c0da08-d238-44dc-8164-a7e413716732-e693/specs/

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/License/License.tsx [hunk 1]
```diff
@@ -49,23 +49,22 @@ const styles = (theme: Theme) =>
...
       padding: "20px 52px 0px 28px",
-      background:
-        "transparent linear-gradient(180deg, #ffffff 0%, #d6e1e8 100%) 0% 0% no-repeat padding-box",
+      background: "#032F51",
       boxShadow: "0px 3px 7px #00000014",
       "& h2": {
-        color: "#000",
-        marginBottom: "50px",
+        color: "#FFF",
+        marginBottom: "67px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:56]: Please remove the px & leave it as int
```suggestion
        marginBottom: 67,
```

File: portal-ui/src/screens/Console/License/License.tsx [hunk 2]
```diff
@@ -81,26 +80,29 @@ const styles = (theme: Theme) =>
...
     currentPlan: {
       fontWeight: 700,
-      background:
-        "transparent linear-gradient(90deg, #073052 0%, #081C42 100%) 0% 0% no-repeat padding-box",
-      boxShadow: "0px 3px 7px #00000014",
-      color: "#fff",
+      background: "#D5DDE5",
+      borderRadius: "3px 3px 0px 0px",
+      color: "#121212",
+      padding: "8px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:100]: Please leave it as int
```suggestion
      padding: 8,
```

File: portal-ui/src/screens/Console/License/License.tsx [hunk 3]
```diff
@@ -110,18 +112,17 @@ const styles = (theme: Theme) =>
...
       padding: "0% 15%",
-      color: "#474747",
     },
     detailsCapacityMin: {
       fontSize: 10,
     },
     itemContainer: {
       height: 36,
-      borderTop: "1px solid #e5e5e5",
+      // borderTop: "1px solid #e5e5e5",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:121]: Please remove comment

File: portal-ui/src/screens/Console/License/License.tsx [hunk 4]
```diff
@@ -110,18 +112,17 @@ const styles = (theme: Theme) =>
...
     },
     itemContainerDetail: {
       height: 48,
-      borderTop: "1px solid #e5e5e5",
+      // borderTop: "1px solid #e5e5e5",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:125]: Please remove comment

File: portal-ui/src/screens/Console/License/License.tsx [hunk 5]
```diff
@@ -247,6 +263,14 @@ const License = ({ classes, operatorMode }: ILicenseProps) => {
...
       })
       .catch((err: any) => {
+        // setLicenseInfo({
+        //   account_id: 0,
+        //   email: "asdasd",
+        //   expires_at: "asd",
+        //   organization: "asd",
+        //   storage_capacity: 0,
+        //   plan: "enterprise",
+        // });
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:273]: We can remove this

File: portal-ui/src/screens/Console/License/License.tsx [hunk 6]
```diff
@@ -435,14 +460,24 @@ const License = ({ classes, operatorMode }: ILicenseProps) => {
...
+                </a>
               </Paper>
             )}
           </Grid>
           <Grid
             item
             xs={12}
             className={classes.container}
-            style={{ padding: "0px 33px" }}
+            style={{ padding: "23px 33px" }}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/License.tsx:480]: Please try to add this as a class

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**